### PR TITLE
TECH Bug while receiving non text/json response content-type

### PR DESCRIPTION
**Pull request by PerrinStephane**
Parse json is need be

### CODE REVIEW

**Review by Perados — APPROVED** [review]
I wanted to fix this long time ago! Most of the messages caused by this were warnings, not errors, I think. :trollface:

File: pymarsys/connections.py [hunk 1]
```diff
@@ -168,4 +168,6 @@ def __init__(self, username, secret, uri=EMARSYS_URI):
                         await response.text()
                     )
                 )
-            return await response.json()
+            if response.headers['Content-Type'] is 'text/json':
```
Comment (Perados) [inline pymarsys/connections.py]: Use `==` to compare strings, not `is`. `is` compares the `id` of the object, not the real value. Strings are optimised in Python, so your strings will probably have the same `id` in this situation, but still, `==` would be safer here IMO.

https://stackoverflow.com/questions/2988017/string-comparison-in-python-is-vs

**Review by Perados — CHANGES_REQUESTED** [review]
:trollface:

**Review by Tehem — APPROVED** [review]
LGTM

**Pull request merged**